panda/src/audiotraits/config_openalAudio.cxx

```diff
@@ -30,6 +30,23 @@ ConfigVariableString openal_device
  PRC_DESC("Specify the OpenAL device string for audio playback (no quotes).  If this "
           "is not specified, the OpenAL default device is used."));
 
+ConfigVariableInt openal_buffer_delete_retries
+("openal-buffer-delete-retries", 5,
+ PRC_DESC("If deleting a buffer fails due to still being in use, the OpenAL "
+          "sound plugin will wait a moment and retry deletion, with an "
+          "exponentially-increasing delay for each try.  This number "
+          "specifies how many repeat tries (not counting the initial try) "
+          "should be made before giving up and raising an error."));
+
+ConfigVariableDouble openal_buffer_delete_delay
+("openal-buffer-delete-delay", 0.001,
```

Comment:
Have you experimented to see which value is enough here?  I don't feel strongly about this at all, but if 0.2 ms is good enough for most cases, I'd use that as a starting point.

Reply:
I'll do some experiments before merging this down. I have a working minimal test case that demonstrates the bug. I'll have it run some timing loops to determine the average-case and worst-case buffer unlocks.